gemrb/core/Scriptable/Actor.cpp

```diff
@@ -925,7 +925,7 @@ bool Actor::ApplyKit(bool remove, ieDword baseclass, int diff)
 			if (multiclass & msk) {
 				max = GetLevelInClass(i);
 				// don't apply/remove the old kit clab if the kit is disabled
-				if (i == kitclass && !IsDualInactive()) {
+				if (i == kitclass && !IsKitInactive()) {
```

Comment:
this inverts the check for cases covered by the previous one.

Reply:
Sorry, I don't comprehend your comment.  The previous what?  What is inverted?

Anyway, this part of code seems to govern updating kit abilities e.g. on level up.  It used to skip inactive duals since their kit must be also inactive, but here I add the aforementioned "rudimentary support for dualing into kits" by checking if it's truly inactive.  Right?

Reply:
What I meant was, if we forget about the extra case now handled. IsKitInactive calls IsDualInactive. But the return value is inverted, so the behaviour for other cases changed.

Reply:
Nope, `IsKitInactive()` returns `false` if `IsDualInactive()` is `false`, so no inversion here.  Notice the exclamation point:
`if (!IsDualInactive()) return false;`

Reply:
Yes, but the call to IsKitInactive is negated too.

Reply:
But the call to `IsDualInactive()` it replaced was also negated!  So there's a triple negation instead of a single negation, which means no change.  Although I now see how this can be confusing.  Should I add a comment somewhere?

Reply:
You're right, sorry. No need for comments, since this is confusing only while looking at the diff.